src/Sql/dbo/Vault/Stored Procedures/Cipher/Cipher_UpdateWithCollections.sql

```diff
@@ -12,8 +12,9 @@
     @DeletedDate DATETIME2(7),
     @Reprompt TINYINT,
     @Key VARCHAR(MAX) = NULL,
-    @CollectionIds AS [dbo].[GuidIdArray] READONLY,
-    @ArchivedDate DATETIME2(7) = NULL
+    @ArchivedDate DATETIME2(7) = NULL,
```

Comment:
🎨 We should be able to drop the old `ArchivedDate` column here and just use `Archives`. (its a `Ciphers_*` sproc and not `CipherDetails_*`)